README.md

```diff
@@ -348,10 +347,21 @@
   ```
 
 * <a name="true-in-cond"></a>
-  Use `true` as the always-match condition of the `cond` special form.
+  Never use other than `true` (for example: `:any`) as the always-match condition of the `cond` special form.
   <sup>[[link](#true-in-cond)]</sup>
 
   ```elixir
+  # Bad
+  cond do
+    char in ?0..?9 ->
+      char - ?0
+    char in ?A..?Z ->
+      char - ?A + 10
+    :any ->
```

Comment:
now I see that `:any` could also be misinterpreted and `:other` is much more better
nice catch!